lib/formatters/sarif.js

```diff
@@ -77,17 +70,13 @@ export default class SarifPrinter {
         };
 
         if (embedFileContents) {
-          try {
```

Comment:
Why does this `try`/`catch` get deleted? Think it's not needed?

Reply:
Ya. I mean I mainly deleted it because we were inline outputting the error and you don't really get any benefit there.